{
  "id": 1734148079947,
  "script": "\n            async function test(a, b) {\n                return Promise.reject(new Error(\"async fail\"));\n            }\n            test(63.0382425037886, 23.977253434457868);\n        ",
  "success": false,
  "error": {
    "message": "Execution error: async fail",
    "stack": "Error: Execution error: async fail\n    at SimpleTestRunner.runTest (/home/ath/Desktop/wgan/simple-solution.js:94:23)\n    at async main (/home/ath/Desktop/wgan/simple-solution.js:163:24)"
  },
  "timestamp": "2024-12-14T03:47:59.948Z"
}